include/boost/math/tools/ulps_plot.hpp

```diff
@@ -433,9 +438,8 @@ template<class F, typename PreciseReal, typename CoarseReal>
 ulps_plot<F, PreciseReal, CoarseReal>::ulps_plot(F hi_acc_impl, CoarseReal a, CoarseReal b,
              size_t samples, bool perturb_abscissas, int random_seed)
 {
-    // This fails for heap allocated types like MPFR.
-    // Got to think on what to do about it.
-    static_assert(sizeof(PreciseReal) >= sizeof(CoarseReal), "PreciseReal must have larger size than CoarseReal");
+    // Use digits10 for this comparison in case the two types have differeing radixes:
+    static_assert(std::numeric_limits<PreciseReal>::digits10 >= std::numeric_limits<CoarseReal>::digits10, "PreciseReal must have higher precision that CoarseReal");
```

Comment:
Yes, I've noticed this problem before for MPFR types; since the `sizeof` operator returns the size of a pointer. Thanks!